github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -109,6 +109,32 @@ presubmits:
             - "/bin/bash"
             - "-ce"
             - "cd images && ./publish_image.sh -b bootstrap quay.io kubevirtci"
+          # docker-in-docker needs privileged mode
+          securityContext:
+            privileged: true
+          resources:
+            requests:
+              memory: "1Gi"
+            limits:
+              memory: "1Gi"
+  - name: build-golang-image
+    always_run: false
+    run_if_changed: "images/golang/.*"
+    decorate: true
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-kubevirtci-quay-credential: "true"
+    cluster: phx-prow
```

Comment:
Do we want to use this cluster for new jobs? I thought this would be turned off at some point?